pkg/sfu/downtrack.go

```diff
@@ -695,9 +643,13 @@ func (d *DownTrack) AdjustAllocation(availableChannelCapacity uint64) (isPausing
 		}
 	}
 
-	// no layer fits in the available channel capacity, disable the track
-	isPausing = !d.bandwidthConstrainedMuted.get()
-	d.bandwidthConstrainedMute(true)
+	if optimalBandwidthNeeded != 0 {
```

Comment:
Checking the optimal needed here to prevent disabling if there are really no tracks in the feed.